### PR TITLE
Bring consistency in backups documentation

### PR DESCRIPTION
**Pull request by GuGuss**
## Why

Closes #{ISSUE_NUMBER}

### CODE REVIEW

File: docs/src/environments/backup.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ file=none
 highlight=false
 ---
 
-1. Navigate to the environment you want to back up.
+1. Navigate to the environment.
```
Comment (CollierCZ) [inline docs/src/environments/backup.md:155]:
```suggestion
1. Navigate to the environment you want to back up.
```

File: docs/src/environments/backup.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ highlight=false
 ### Automate manual backups
 
 You can also automate the process of creating manual backups through [cron jobs](../create-apps/app-reference.md#crons).
-The cron job uses the CLI command to back up the environment.
+The cron job uses the CLI command to run a backup of the environment.
```
Comment (CollierCZ) [inline docs/src/environments/backup.md:165]:
```suggestion
The cron job uses the CLI command to back up the environment.
```

File: docs/src/environments/backup.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Back up environments
+title: Create a backup
```
Comment (CollierCZ) [inline docs/src/environments/backup.md:2]:
```suggestion
title: Back up an environment
```

**Review by AnouckColson — DISMISSED** [review]
Perfect!

**Review by CollierCZ — APPROVED** [review]

**Pull request merged**